python/lsst/sconsUtils/dependencies.py

```diff
@@ -504,7 +504,6 @@ def _tryImport(self, name):
                 else:
                     module.config.addCustomTests(self.customTests)
                 return module
-        state.log.info("Failed to import configuration for optional package '%s'." % name)
```

Comment:
The `_tryImport` method is used for required and optional packages.  The info message doesn't really seem to fit.